internal/hardware/validator_test.go

```diff
@@ -196,21 +198,54 @@ var _ = Describe("Cluster host requirements", func() {
 		details1, details2   models.ClusterHostRequirementsDetails
 	)
 
+	const (
+		prefixedRequirementsEnv = "MYAPP_" + requirementsEnv
+		versionRequirements     = `[{
```

Comment:
Please make it a native Map and encode it into a JSON, so we won't have syntax issues.